bundler/lib/bundler/plugin.rb

```diff
@@ -47,6 +47,32 @@ def install(names, options)
       Bundler.ui.error "Failed to install plugin #{name}: #{e.message}\n  #{e.backtrace.join("\n ")}"
```

Comment:
I was reading this block to see if some of the "uninstallation logic" could/should be reused here, and I'm wondering... where is the `name` variable being defined? Is this actually working/neeed? :thinking:

I know this is not your code, but if you're willing to double check this, I appreciate it.

Reply:
This is needed, but not working. In eb5873fc0e method signature was changed, but this line stayed the same.
I think it is better to fix this in a separate pr.

Reply:
I opened #3484 to track it.